File: Hybrid_Task/Python/encrypt_data.py
number = input("Enter a four-digit number: ")
if int(number) < 1000 or int(number) > 9999:
	print("Invalid input!")

else: 
	inflation = []
	for digits in number :
		inflate = (int(digits) + 7) % 10
		inflation.append(inflate)

	rearrange = []
	rearrange.append(inflation[2])
	rearrange.append(inflation[3])
	rearrange.append(inflation[0])
	rearrange.append(inflation[1])


	encrypted = ""
	for elements in rearrange:
		encrypted += str(elements)
	
	print(encrypted)

File: Hybrid_Task/Javascript/encrypt_data.js
let prompt = require('prompt-sync')();


let number = prompt("Enter a four-digit number: ");

let digits = " ";

let power = 1000;

for(let count = 1; count <= 4; count += 1) {
	let digit = number // power;
	//console.log(digit);
	digits += (digit + 7) % 10;
	//console.log(digits);
	number %= power;
	//console.log(number);
	power *= 0.1;
	//console.log(power);	
}
console.log(digits);


//int[] rearrange = new int[4];
//rearrange[0] = digits[2];
//rearrange[1] = digits[3];
//rearrange[2] = digits[0];
//rearrange[3] = digits[1];

//System.out.print(rearrange[0]);
//System.out.print(rearrange[1]);
//System.out.print(rearrange[2]);
//System.out.print(rearrange[3]);
//}
//}